app/controllers/Home.php

```diff
@@ -101,5 +101,15 @@ public function ubah()
     $this->view('home/update', $data);
     $this->view('templates/footer');
   }
-
+  public function cari(){
+    $data['title'] = 'home';
+    if(isset($_POST)){
+      $key = $_POST;
+      $data['obat'] = $this->model('Obat_model')->getObatByKey($key['keyword']);
```

Comment:
hilangkan warning `Warning: Undefined array key "keyword" in D:\laragon\www\apotek-rsstt\app\controllers\Home.php`